pubspec.yaml

```diff
@@ -17,11 +17,11 @@ dependencies:
   wakelock: ^0.1.4+1
   image_picker: ^0.6.7+1
   dash_chat: ^1.1.8
-  camera:
-    git:
-      url: https://github.com/Renesanse/plugins.git
-      path: packages/camera
-      ref: flash_light
+  # camera:
```

Comment:
You can remove a piece of code when required instead of turning it into a comment. We have GitHub if we need it back.